content_creator.py

```diff
@@ -0,0 +1,33 @@
+from image_loader import ImageLoader
+from quote_loader import QuoteLoader
+from message_sender import MessageSender
+import logging
+
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+logger = logging.getLogger("ContentCreationLogger")
+
+
+class ContentCreator:
+    def __init__(self, quote_api_url, image_api_url, webhook_url, author_name):
+        self.quote_loader = QuoteLoader(quote_api_url)
+        self.image_loader = ImageLoader(image_api_url)
+        self.message_sender = MessageSender(webhook_url)
+        self.author_name = author_name
+
+    def create_content(self):
```

Comment:
It should be a _create_content function, if you are not aiming to call it directly